micropub.php

```diff
@@ -197,6 +235,25 @@ private static function authorize() {
     return NULL;
   }
 
+  private static function return_query($user_id) {
+    switch($_GET['q']) {
+      case 'syndicate-to':
+      // Fallback
+      case 'mp-syndicate-to':
+        status_header(200);
```

Comment:
minor: these two lines will probably apply to all queries, not just `syndicate-to`, so consider moving them above the switch